src/main/java/org/jooq/lambda/Seq.java

```diff
@@ -3397,6 +3397,30 @@ public R next() {
     }
 
     /**
+     * Chunk a Stream based on a predicate
+     * <p>
+     * <code><pre>
+     * // ((1,1,2), (2), (3,4))
+     * Seq.of(1, 1, 2, 2, 3, 4).chunked(n -> n %2 == 0)
```

Comment:
I was also thinking an enum would be the appropriate solution for inclusion / exclusion. It makes sense to work for all four.

>2. The beginning chunk ((1, 1), (2), (2, 3), (4))

Should we assume the beginning chunk starts true? or should this actually return `((2), (2, 3), (4))` skipping anything before the first chunk.

> 4. No chunk

Would this be excluding the matched element? `((1, 1), (), (3))`